source/hic_hal/stm32/stm32h743ii/IO_Config.h

```diff
@@ -84,117 +79,112 @@ COMPILER_ASSERT(DAPLINK_HIC_ID == DAPLINK_HIC_ID_STM32H743II);
-#define UDC0_BOOT_L_DIR_PORT           GPIOE
-#define UDC0_BOOT_L_DIR_PIN            GPIO_PIN_15
-#define UDC0_BOOT_L_DIR_PIN_Bit        15
+#define UDC0_BOOT_L_PORT                GPIOE
+#define UDC0_BOOT_L_PIN                 GPIO_PIN_7
+#define UDC0_BOOT_L_PIN_Bit             7
+#define UDC0_BOOT_L_DIR_PORT            GPIOE
+#define UDC0_BOOT_L_DIR_PIN             GPIO_PIN_15
+#define UDC0_BOOT_L_DIR_PIN_Bit         15
 
 //MCU_BUT_USR0_UDC_L, BUT_USR0_DIR
-#define UDC0_BUTTON_L_PORT             GPIOE
-#define UDC0_BUTTON_L_PIN              GPIO_PIN_8
-#define UDC0_BUTTON_L_PIN_Bit          8
-#define UDC0_BUTTON_L_DIR_PORT         GPIOF
-#define UDC0_BUTTON_L_DIR_PIN          GPIO_PIN_2
-#define UDC0_BUTTON_L_DIR_PIN_Bit      2
+#define UDC0_BUTTON_L_PORT              GPIOE
+#define UDC0_BUTTON_L_PIN               GPIO_PIN_8
+#define UDC0_BUTTON_L_PIN_Bit           8
+#define UDC0_BUTTON_L_DIR_PORT          GPIOF
+#define UDC0_BUTTON_L_DIR_PIN           GPIO_PIN_2
+#define UDC0_BUTTON_L_DIR_PIN_Bit       2
 
 //JTAG1_MCU_UDC_RESET, JTAG1_RESET_DIR
-#define UDC1_RST_PORT               GPIOI
-#define UDC1_RST_PIN                GPIO_PIN_9
-#define UDC1_RST_PIN_Bit            9
-#define UDC1_RST_DIR_PORT           GPIOB
-#define UDC1_RST_DIR_PIN            GPIO_PIN_4
-#define UDC1_RST_DIR_PIN_Bit        4
+#define UDC1_RST_PORT                   GPIOI
+#define UDC1_RST_PIN                    GPIO_PIN_9
+#define UDC1_RST_PIN_Bit                9
+#define UDC1_RST_DIR_PORT               GPIOB
+#define UDC1_RST_DIR_PIN                GPIO_PIN_4
+#define UDC1_RST_DIR_PIN_Bit            4
 
 //MCU_BOOT1_UDC, BOOT1_DIR
-#define UDC1_BOOT_PORT               GPIOE
-#define UDC1_BOOT_PIN                GPIO_PIN_1
-#define UDC1_BOOT_PIN_Bit            1
-#define UDC1_BOOT_DIR_PORT           GPIOE
-#define UDC1_BOOT_DIR_PIN            GPIO_PIN_13
-#define UDC1_BOOT_DIR_PIN_Bit        13
+#define UDC1_BOOT_PORT                  GPIOE
+#define UDC1_BOOT_PIN                   GPIO_PIN_1
+#define UDC1_BOOT_PIN_Bit               1
+#define UDC1_BOOT_DIR_PORT              GPIOE
+#define UDC1_BOOT_DIR_PIN               GPIO_PIN_13
+#define UDC1_BOOT_DIR_PIN_Bit           13
 
 //MCU_BUT_USR1_UDC, BUT_USR1_DIR
-#define UDC1_BUTTON_PORT             GPIOE
-#define UDC1_BUTTON_PIN              GPIO_PIN_3
-#define UDC1_BUTTON_PIN_Bit          3
-#define UDC1_BUTTON_DIR_PORT         GPIOE
-#define UDC1_BUTTON_DIR_PIN          GPIO_PIN_14
-#define UDC1_BUTTON_DIR_PIN_Bit      14
-
-// ehassman
-// From MX_Device.h
-/*-------------------------------- I2C1       --------------------------------*/
+#define UDC1_BUTTON_PORT                GPIOE
+#define UDC1_BUTTON_PIN                 GPIO_PIN_3
+#define UDC1_BUTTON_PIN_Bit             3
+#define UDC1_BUTTON_DIR_PORT            GPIOE
+#define UDC1_BUTTON_DIR_PIN             GPIO_PIN_14
+#define UDC1_BUTTON_DIR_PIN_Bit         14
+
+/* The I2C logic was written by an intern. I2C_STM32H7xx.c is generated through
```

Comment:
Are you implying something with this comment...? I don't think it matters if the I2C logic was written by an intern, maybe delete that sentence.